repos/fdbt-site/src/pages/api/managePassengerGroup.ts

```diff
@@ -35,7 +39,7 @@ export const formatRequestBody = (req: NextApiRequestWithSession): GroupPassenge
     });
 
     return {
-        id,
+        id: id,
```

Comment:
is there a reason for removing the shorthand here?

Reply:
Nope, just from some refactoring I did and then reverted left this lying around